Sources_v3/Workers/EventNotificationCenter.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+class EventNotificationCenter {
```

Comment:
I wonder, if it made sense to actually make this a subclass of `NotificationCenter`? When using it, you will do `eventNotificationCenter.notificationCenter` all the time, anyway.

Reply:
done in 9ccf3e7d5bc8ffbd0021704e53abe9deed9320bb